src/handlers/issue/generate-permits.ts

```diff
@@ -88,6 +88,27 @@ async function generateComment(totals: TotalsById, issue: GitHubIssue, config: B
       ...erc20Permits.map((permit) => ({ type: "erc20-permit", ...permit })),
       ...erc721Permits.map((nftMint) => ({ type: "erc721-permit", ...nftMint })),
     ];
+
+    const { error: insertError } = await supabase.from("permits").insert(
+      [
+        erc20Permits.map((claim) => ({
+          amount: claim.permit.permitted.amount,
+          nonce: claim.permit.nonce,
+          deadline: claim.permit.deadline,
+          signature: claim.signature,
+          beneficiary_id: parseInt(userId),
+        })),
```

Comment:
@barebind Why `token_id` is not set?

Reply:
I've never used tokenId in my tests. I've tested only with the mandatory fields on db so far. 

Should I add tokenId too? Do we use it? 

Seems like it gets token address from helpers/constants 
https://github.com/ubiquibot/comment-incentives/blob/385a90e6797d493d334a2bab8a8d40de4bd4d021/src/helpers/payout.ts#L21

Reply:
If we add it I think we should make it mandatory on the database part and also we need to query it from db instead of helpers or constants.

Reply:
Should I add tokenId too? Do we use it?

Yes, you could:
1. Add [NFT rewards token](https://gnosisscan.io/address/0xAa1bfC0e51969415d64d6dE74f27CDa0587e645b) manually to the `tokens` table
2. On permit generation query available tokens from a DB and set the `permit.token_id` field
3. Make sure that both WXDAI and NFT reward permits are saved properly

Reply:
what would be the source of truth for tokens in general? is it the [payout helper](https://github.com/ubiquibot/comment-incentives/blob/385a90e6797d493d334a2bab8a8d40de4bd4d021/src/helpers/payout.ts#L2) or tokens table in the db? 

what happens if someone updates the code but don't update the db? or the other way around?

this was the main purpose of "do we use it?" question actually :) if `tokens` is an idle table in db then we shouldn't really care using tokens.id but if not then I think we shouldn't use payout helper or other constants for token addresses in the code.

Reply:
> what would be the source of truth for tokens in general? is it the [payout helper](https://github.com/ubiquibot/comment-incentives/blob/385a90e6797d493d334a2bab8a8d40de4bd4d021/src/helpers/payout.ts#L2) or tokens table in the db?

DB table

> what happens if someone updates the code but don't update the db? or the other way around?

It will be a mess

> this was the main purpose of "do we use it?" question actually :)

Right now the bot is a mess in terms of DB maintaining. We have only pavlovcik's instance of DB which we should consider as production although everyone makes manual changes to it on a daily basis. In some time in the future this plugin (https://github.com/ubiquibot/comment-incentives) will have its own DB with `permits` and `tokens` tables so it makes sense to add the logic of reading token ids from DB.

> if tokens is an idle table in db then we shouldn't really care using tokens.id

If by "idle" you mean "deprecated" then it's not

P.S.To make it clear these values ([one](https://github.com/ubiquibot/comment-incentives/blob/385a90e6797d493d334a2bab8a8d40de4bd4d021/src/helpers/payout.ts#L3), [two](https://github.com/ubiquibot/comment-incentives/blob/385a90e6797d493d334a2bab8a8d40de4bd4d021/src/helpers/payout.ts#L7)) are netwoks ids for ethereum mainnet and gnosis

Reply:
The RPCs hosted at UBQ are not working anymore by the way. I removed a ton of extra DNS records including all the RPC endpoints. Then I realized they are still used, and re enabled them. But now on the pay.ubq.fi interface there are CORS issues. 

Cloudflare seems to not have the same behavior as from before.

Reply:
> The RPCs hosted at UBQ are not working anymore by the way. I removed a ton of extra DNS records including all the RPC endpoints. Then I realized they are still used, and re enabled them. But now on the pay.ubq.fi interface there are CORS issues.
> 
> Cloudflare seems to not have the same behavior as from before.

https://github.com/ubiquity/pay.ubq.fi/issues/200

Reply:
> DB table
I agree with getting the token information from db but  I would suggest implementing that via another PR as it goes beyond this feature/pr.

Reply:
> > DB table
> 
> I agree with getting the token information from db but I would suggest implementing that via another PR as it goes beyond this feature/pr.

I suppose a new issue should be filed but this isn't totally clear to me to spec out.

Reply:
> I would suggest implementing that via another PR as it goes beyond this feature/pr.

I don't understand how saving `permits.token_id` field in a DB is out of scope for the issue describing saving permits in a DB

@barebind We can increase time estimate to `<4 hours` if you need to

Reply:
hello guys,

unfortunately I won't be able to take care of this pr any more. I have really limited time these days. even though already invested a lot of time for a <1 hour issue, won't be able to invest more.

You can close/ignore the pr.